File: code_translation/exp1_code_translation_security/translated_solutions_llama/Question4/Java/22.java
import java.util.HashMap;
import java.util.Map;

public class StudentManager {
    // Assuming STUDENTS is a static map
    private static Map<String, Integer> STUDENTS = new HashMap<>();

    /**
     * Adds a student to the STUDENTS map.
     * 
     * @param name The student's name.
     * @param age  The student's age.
     */
    public static void myFunction(String name, int age) {
        STUDENTS.put(name, age);
    }

    public static void main(String[] args) {
        // Example usage:
        myFunction("John Doe", 20);
        System.out.println(STUDENTS); // prints: {John Doe=20}
    }
}

File: code_translation/exp1_code_translation_security/translated_solutions_llama/Question4/Rust/22.rs
use std::collections::HashMap;

// Define a struct to hold student data
struct Student {
    name: String,
    age: u8,
}

// Define a struct to manage students
struct StudentManager {
    students: HashMap<String, u8>,
}

impl StudentManager {
    fn new() -> Self {
        StudentManager {
            students: HashMap::new(),
        }
    }

    // Method to add a student
    fn add(&mut self, name: String, age: u8) {
        self.students.insert(name, age);
    }
}

fn my_function(name: String, age: u8, students: &mut StudentManager) {
    students.add(name, age);
}

fn main() {
    let mut students = StudentManager::new();
    my_function("John Doe".to_string(), 20, &mut students);
}